pkg/apiserver/registry/networkpolicy/clustergroupmember/rest.go

```diff
@@ -48,11 +48,16 @@ func (r *REST) New() runtime.Object {
 	return &controlplane.ClusterGroupMembers{}
 }
 
-func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
+func (r *REST) Get(ctx context.Context, name string, options runtime.Object) (runtime.Object, error) {
 	groupMembers, ipBlocks, err := r.querier.GetGroupMembers(name)
 	if err != nil {
 		return nil, errors.NewInternalError(err)
 	}
+	// Retrieve options used for pagination, default to 0 when all members are returned.
+	getOptions, ok := options.(*controlplane.PaginationGetOptions)
+	if !ok {
+		getOptions = &controlplane.PaginationGetOptions{Page: 0, Limit: 0}
+	}
```

Comment:
IMO, if `ok` is false, it could be treated as an internal server error. It should not be possible based on the K8s code.

Reply:
Updated the error handling cases including this `ok` and `limit | page < 0`.